Adding Seifart-2015-414 conceptlist

MuffinLinwist opened the pull request:
# Pull request checklist

- [x] add new concept list
- [x] add new metadata
- [ ] add new Concepticon concept sets
- [ ] checked whether the new concept(s) can be applied to existing lists with `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

...

  This is the requested concept list extracted from the [lexibank/seifartecheverriboran](https://github.com/lexibank/seifartecheverriboranl) dataset.

Tarotis commented:
Thanks for adding this list.

@chrzyki Could we ask you to moderate this PR? I'd provide the first review, and @AnnikaTjuka already accepted to take over the second review.

@MuffinLinwist Please remember that when opening the PR, it is better not to tag anybody specifically. Somebody will moderate the PR (check for basic things) and then assign the reviewers. This way, there is less confusion about who will actually work on a PR, and the tasks will get distributed by the Concepticon editors. This also assures a better overview on the editor's site about who is already doing multiple reviews and who could help.

MuffinLinwist commented:
> Thanks for adding this list.
> 
> @chrzyki Could we ask you to moderate this PR? I'd provide the first review, and @AnnikaTjuka already accepted to take over the second review.
> 
> @MuffinLinwist Please remember that when opening the PR, it is better not to tag anybody specifically. Somebody will moderate the PR (check for basic things) and then assign the reviewers. This way, there is less confusion about who will actually work on a PR, and the tasks will get distributed by the Concepticon editors. This also assures a better overview on the editor's site about who is already doing multiple reviews and who could help.

Thanks for the comment. I already modified the opening message :)

chrzyki commented:
> Thanks for adding this list.
> 
> @chrzyki Could we ask you to moderate this PR? I'd provide the first review, and @AnnikaTjuka already accepted to take over the second review.
> 
> @MuffinLinwist Please remember that when opening the PR, it is better not to tag anybody specifically. Somebody will moderate the PR (check for basic things) and then assign the reviewers. This way, there is less confusion about who will actually work on a PR, and the tasks will get distributed by the Concepticon editors. This also assures a better overview on the editor's site about who is already doing multiple reviews and who could help.

Happy to moderate! Thank you @MuffinLinwist for preparing the list and @Tarotis for doing the first and @AnnikaTjuka the second review.

The repository for this concept list is https://github.com/lexibank/seifartecheverriboran, correct?

One small thing for now: We've got

```
Journal={LIAMES: Línguas Indígenas Americanas},
```

and

```
journal={LIAMES}
```

for stating naming this particular journal and I'd suggest the full (i.e. former) form.

Thanks again and I'll have a closer look after the review(s).

MuffinLinwist commented:
> > Thanks for adding this list.
> > @chrzyki Could we ask you to moderate this PR? I'd provide the first review, and @AnnikaTjuka already accepted to take over the second review.
> > @MuffinLinwist Please remember that when opening the PR, it is better not to tag anybody specifically. Somebody will moderate the PR (check for basic things) and then assign the reviewers. This way, there is less confusion about who will actually work on a PR, and the tasks will get distributed by the Concepticon editors. This also assures a better overview on the editor's site about who is already doing multiple reviews and who could help.
> 
> Happy to moderate! Thank you @MuffinLinwist for preparing the list and @Tarotis for doing the first and @AnnikaTjuka the second review.
> 
> The repository for this concept list is https://github.com/lexibank/seifartecheverriboran, correct?
> 
> One small thing for now: We've got
> 
> ```
> Journal={LIAMES: Línguas Indígenas Americanas},
> ```
> 
> and
> 
> ```
> journal={LIAMES}
> ```
> 
> for stating naming this particular journal and I'd suggest the full (i.e. former) form.
> 
> Thanks again and I'll have a closer look after the review(s).

Thanks for noticing this! It's done :)

Tarotis commented on concepticondata/conceptlists.tsv:
```diff
@@ -429,4 +429,5 @@ Capell-1956-558	Capell, Arthur	1956		558	basic	English	North Australian language
-Girard-1971-559	Girard, Victor	1971		559		Spanish	Tacanan languages		Girard1971		This list is extracted from a study for the classification of the Panoan and Tacanan families. The study provides cognates for Tacanan phonology, morphology, and lexicon. It also presents a synchronic description and classification of Tacanan languages.	50-138	
\ No newline at end of file
+Girard-1971-559	Girard, Victor	1971		559		Spanish	Tacanan languages		Girard1971		This list is extracted from a study for the classification of the Panoan and Tacanan families. The study provides cognates for Tacanan phonology, morphology, and lexicon. It also presents a synchronic description and classification of Tacanan languages.	50-138	
+Seifart-2015-414	Seifart, Frank and Echeverri, Juán Álvaro	2015		414		English	Bora and Muinane		Seifart2015		This list contains concepts used for a reconstruction of the proto Bora-Muinane phonological system and of 415 word forms. It also reassess the genealogical relationship of Bora-Muinane with Witoto, Ocaina, and Nonuya.	279-311	
```
It is "Juan", not Juán"

Tarotis replied:
"It also reassess the genealogical relationship of Bora-Muinane with Witoto, Ocaina, and Nonuya"

No, it does not.

Tarotis commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-62	62	cooking pot	3653	COOKING POT	82
+Seifart-2015-414-63	63	cultivated clearing	212	FIELD	83
+Seifart-2015-414-64	64	village	930	VILLAGE	84
+Seifart-2015-414-65	65	path,trail	2457	PATH OR ROAD	85
```
We can probably map this to the narrower PATH

MuffinLinwist replied:
It was like that on the automatic linking but I change it based on previous mappings.

Tarotis commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-65	65	path,trail	2457	PATH OR ROAD	85
+Seifart-2015-414-66	66	fish net	388	FISHNET	86
+Seifart-2015-414-67	67	fish hook	1074	FISHHOOK	87
+Seifart-2015-414-68	68	steel axe			88
```
I think we could map this to AXE

Tarotis commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-77	77	puma	1891	PUMA	99
+Seifart-2015-414-78	78	armadillo	1885	ARMADILLO	100
+Seifart-2015-414-79	79	dog	2009	DOG	101
+Seifart-2015-414-80	80	red deer			102
```
We could think about adding RED DEER as a sub-concept of DEER. Please open an issue about this

See https://concepticon.clld.org/parameters/1936

MuffinLinwist replied:
This is now pending for approval on the PR #1339.

chrzyki replied:
Thank you for mapping this, approved.

Tarotis commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-80	80	red deer			102
+Seifart-2015-414-81	81	bat	1793	BAT	103
+Seifart-2015-414-82	82	otter	15	OTTER	104
+Seifart-2015-414-83	83	cebus,monkey	439	CEBUS MONKEY	105
```
Is the comma in "cebus, monkey" correct?

MuffinLinwist replied:
It appears like this on the original `raw file` on the dataset.

Tarotis commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-114	114	spider	843	SPIDER	147
+Seifart-2015-414-115	115	jigger (tunga penetrans)	438	CHIGGOE (JIGGER FLEA)	148
+Seifart-2015-414-116	116	bush	1683	BUSH	149
+Seifart-2015-414-117	117	open grassland	260	SAVANNA	150
```
This should be GRASSLAND (https://concepticon.clld.org/parameters/2819) instead

Tarotis commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-128	128	cotton	1850	COTTON	166
+Seifart-2015-414-129	129	gourd	411	GOURD	167
+Seifart-2015-414-130	130	yam	410	YAM	168
+Seifart-2015-414-131	131	annatto	349	ACHIOTE	170
```
I don't think this fits

MuffinLinwist replied:
What do you mean? Annatto is achiote in fact.

Tarotis replied:
I thought that ACHIOTE would refer to the tree, not the condiment, but I see that there are many previous mappings. So you are right.

Tarotis commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-142	142	first	728	FIRST	191
+Seifart-2015-414-143	143	last	1529	LAST (FINAL)	192
+Seifart-2015-414-144	144	drum	908	DRUM	194
+Seifart-2015-414-145	145	ear ornament			196
```
There are many entries in Concepticon regarding "ear ornament". Please open an issue whether those should be mapped under "earring", or whether we should add a new concept "ear ornament".

MuffinLinwist replied:
This was already responded in #1336. We should map this gloss as EARRING. With the last commit, this was implemented.

Tarotis commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-212	212	belly	1251	BELLY	272
+Seifart-2015-414-213	213	liver	1224	LIVER	273
+Seifart-2015-414-214	214	drink	1401	DRINK	274
+Seifart-2015-414-215	215	eat (non meat)			275
```
This is the most generic "eat" in the list, compared to "eat meat" and "eat fruit". If @AnnikaTjuka agrees, we could map this to EAT, but I'd like to have a second opinion on this

AnnikaTjuka replied:
I like the idea that there is a special word for eating meat and not the other way round. We have to indicate if someone eats vegetarian. I'd map it for now.

Tarotis commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-293	293	diminutive			378
+Seifart-2015-414-294	294	big river			379
+Seifart-2015-414-295	295	longhouse			380
+Seifart-2015-414-296	296	stone axe	677	AXE	382
```
We can unite this with "steel axe", since they present the same forms. See note in the original concept list

Tarotis commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-291	291	rope	1218	ROPE	374
+Seifart-2015-414-292	292	year	1226	YEAR	375
+Seifart-2015-414-293	293	diminutive			378
+Seifart-2015-414-294	294	big river			379
```
Same as "river" (=49)

Tarotis commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-298	298	deer	1936	DEER	384
+Seifart-2015-414-299	299	macaw (blue)	1630	MACAW	385
+Seifart-2015-414-300	300	ant (conga)	587	ANT	386
+Seifart-2015-414-301	301	bitter manioc (plant)			387
```
This is CASSAVA

AnnikaTjuka replied:
But CASSAVA was mapped previously. Can we merge the two?

MuffinLinwist replied:
In past lists, this is unmapped. Should I open an issue about that?

AnnikaTjuka replied:
It seems that the `cassava (bread)` is usually the unmapped one so I'd do the same in this list, i.e., map `bitter manioc (plant)` and unmap `cassava (bread)`.

Tarotis replied:
I agree with Annika here

MuffinLinwist replied:
Perhaps `cassava (bread)` here could be mapped as [407 MANIOC BREAD](https://concepticon.clld.org/parameters/407)?

Tarotis replied:
Good catch! Seems to fit perfectly

MuffinLinwist replied:
Ok, I kept them separately and mapped them accordingly.

Tarotis commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-301	301	bitter manioc (plant)			387
+Seifart-2015-414-302	302	push	1452	PUSH	388
+Seifart-2015-414-303	303	find	719	FIND	389
+Seifart-2015-414-304	304	get ill			390
```
We can map this under 

  | BECOME SICK
-- | --


[BECOME SICK](https://concepticon.clld.org/parameters/2327)

Tarotis commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-304	304	get ill			390
+Seifart-2015-414-305	305	squeeze manioc dough			391
+Seifart-2015-414-306	306	germinate			392
+Seifart-2015-414-307	307	scream	3809	SCREAM (PRODUCE A CRY)	394
```
Are we sure this is the verb?

Tarotis replied:
I can ask the authors about it to reassure

MuffinLinwist replied:
I think better to ask for this too.

Tarotis replied:
Mapping is correct

Tarotis commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-327	327	roast	1962	ROAST (SOMETHING)	435
+Seifart-2015-414-328	328	bring	692	BRING	437
+Seifart-2015-414-329	329	pay a visit	1377	VISIT	440
+Seifart-2015-414-330	330	grassland	2819	GRASSLAND	441
```
Same as "open grassland", please unite concepts

Tarotis commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-335	335	bitter	887	BITTER	447
+Seifart-2015-414-336	336	fast	1631	FAST	448
+Seifart-2015-414-337	337	blind	1653	BLIND	450
+Seifart-2015-414-338	338	dock,port			451
```
We can map this as PORT

Tarotis commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-383	383	inga edulis	3981	GUAVA	518
+Seifart-2015-414-384	384	ceramic saucepan			519
+Seifart-2015-414-385	385	uvilla fruit (pourouma cecropiaefolia)			520
+Seifart-2015-414-386	386	rifle	3881	PISTOL	521
```
Rifle is generally unmapped. We could also consider this as new concept

AnnikaTjuka replied:
We discussed this before but decided to leave it for now. https://github.com/concepticon/concepticon-data/issues/1234

MuffinLinwist replied:
I add this at the list of issues to be opened in Concepticon.

MuffinLinwist replied:
Then, I'll just unmap the gloss on this list.

Tarotis reviewed (changes requested):
Thanks for adding this list. I have made a couple of suggestions for edits.

In some cases, I think we can merge concepts, since the authors provide the same form anyway, and even give comments like `(Note =49,311)` (as for RIVER and BIG RIVER). In two other cases, I will ask the authors for clarification.

MuffinLinwist commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-117	117	open grassland	260	SAVANNA	150
+Seifart-2015-414-118	118	hill	614	HILL	151
+Seifart-2015-414-119	119	tree	906	TREE	152
+Seifart-2015-414-120	120	leaf	628	LEAF	153
+Seifart-2015-414-121	121	tree leaf	628	LEAF	154
```
So the way to go over this is to have just one single entry:

Seifart-2015-414-120	120	leaf,tree leaf	628	LEAF	153,154

?

Tarotis replied:
Yes

AnnikaTjuka commented on concepticondata/conceptlists.tsv:
```diff
@@ -429,4 +429,5 @@ Capell-1956-558	Capell, Arthur	1956		558	basic	English	North Australian language
-Girard-1971-559	Girard, Victor	1971		559		Spanish	Tacanan languages		Girard1971		This list is extracted from a study for the classification of the Panoan and Tacanan families. The study provides cognates for Tacanan phonology, morphology, and lexicon. It also presents a synchronic description and classification of Tacanan languages.	50-138	
\ No newline at end of file
+Girard-1971-559	Girard, Victor	1971		559		Spanish	Tacanan languages		Girard1971		This list is extracted from a study for the classification of the Panoan and Tacanan families. The study provides cognates for Tacanan phonology, morphology, and lexicon. It also presents a synchronic description and classification of Tacanan languages.	50-138	
+Seifart-2015-414	Seifart, Frank and Echeverri, Juán Álvaro	2015		414		English	Bora and Muinane		Seifart2015		This list contains concepts used for a reconstruction of the proto Bora-Muinane phonological system and of 415 word forms. It also reassess the genealogical relationship of Bora-Muinane with Witoto, Ocaina, and Nonuya.	279-311	
```
I think we usually list the languages in the `target/source language` column with a comma instead of "and". Also, change "reassess" to "reassesses".

AnnikaTjuka commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-174	174	bad	1292	BAD	225
+Seifart-2015-414-175	175	white	1335	WHITE	226
+Seifart-2015-414-176	176	black	163	BLACK	227
+Seifart-2015-414-177	177	go!			228
```
These cases are a bit strange and I noticed that some lists are mapped go! to GO and come! to COME. Since the verbs don't seem to appear anywhere else on the list, I'd map them, but could you open an issue where we discuss these cases?

MuffinLinwist replied:
All of them do appear in an infinitive form in the list. But I can open an issue for us to discuss the cases where they don't.

Tarotis replied:
there is "come" in line 227. But "go" does seem to be missing indeed

MuffinLinwist replied:
So, I'll map `go!` with [695 GO](https://concepticon.clld.org/parameters/695), leave the rest unmapped, and open an issue about these cases.

MuffinLinwist replied:
There's already a response on #1337 on this topic from @LinguList.

AnnikaTjuka commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-193	193	stream	947	STREAM	250
+Seifart-2015-414-194	194	pebbles	961	PEBBLES	251
+Seifart-2015-414-195	195	path	2252	PATH	253
+Seifart-2015-414-196	196	shelter	1678	HUT	255
```
Not all glosses of "shelter" are mapped to HUT so I'd unmap here.

AnnikaTjuka commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-393	393	miriti palm (mauritia flexuosa)	3811	MIRITI PALM	530
+Seifart-2015-414-394	394	pillars (of roundhouse)			532
+Seifart-2015-414-395	395	lepidocaryum tenue palm (for thatching roofs)			533
+Seifart-2015-414-396	396	eat fruit			534
```
Could someone do a study on EAT colexifications with different kinds of food? Or is there already one? I've never seen this distinction in a concept list before.

MuffinLinwist replied:
There is but just [Kaufman-1973-1028-775](https://concepticon.clld.org/values/Kaufman-1973-1028-775).

AnnikaTjuka reviewed (commented):
Thanks for adding the list, @MuffinLinwist! I added a few comments.

MuffinLinwist commented:
@Tarotis, @AnnikaTjuka, and @chrzyki, thanks for your first round of observations. The new version of the PR addresses all of your remarks. We are only waiting on confirmation from the authors of the list for some of the doubts here.

Tarotis commented:
Since we have merged some of the concepts, the new number of concepts is now at 410, not 414. Please fix all the files that refer to this number. I will then make the final review.

MuffinLinwist commented:
@Tarotis, the changes are done for you to do the final review.

Tarotis commented:
Thanks. I'll do the review once I have received the answers of the authors regarding the concepts. Usually they respond to me quite quickly

chrzyki commented:
Thank you all for preparing this list! Current status is that @Tarotis is still waiting for feedback by the authors, correct? Once that is done, @AnnikaTjuka and I can have another look at the list?

AnnikaTjuka reviewed (approved):
I'll be on holiday next week, so I'm approving the PR now. All my suggestions were implemented.

Tarotis commented on concepticondata/conceptlists/Seifart-2015-414.tsv:
```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-279	279	think	2271	THINK	360
+Seifart-2015-414-280	280	sing	1261	SING	361
+Seifart-2015-414-281	281	smell	2124	SMELL	362
+Seifart-2015-414-282	282	blow	175	BLOW (OF WIND)	365
```
This should be mapped as the verb "BLOW (WITH MOUTH)" @MuffinLinwist

MuffinLinwist commented:
@chrzyki and @Tarotis, with the answers from the authors provided by @Tarotis, this PR has all the requested changes implemented and the pending topics resolved. The status of the PR is still waiting for @Tarotis to check if all of his suggestions were implemented and the PR is ready to be merged.

Tarotis reviewed (approved):
Thanks! From my side, we are good to go.

MuffinLinwist commented:
I adapted the changes on the #1339 PR and mapped the `red deer` gloss here. Since, we have all the approvals and all the checks have passed, I'll go ahead and merge the PR.
